cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/SpUtils.kt

```diff
@@ -121,3 +118,6 @@ internal fun userConsents(
             }
     )
 }
+
+internal fun SpConfig.isIncluded(campaign: CampaignType) =
```

Comment:
Actually, from my pov, there are 2 ways we can store those ext functions:
1. store them by type in each file accordingly (meaning that for example if we create extension function for `SpConfig` class, then we create file `SpConfigExt` and add all the ext functions there)
2. store them by logical affiliation (meaning that we have ext functions that are being used only in a certain flow/layer)

Since we don't have a very strict layer separation, I think the first way is the best one for us